ddp-workspace/projects/ddp-dsm-ui/src/app/activity-data/models/activity-definition.model.ts

```diff
@@ -6,16 +6,20 @@ export class ActivityDefinition {
     public activityCode: string,
     public activityName: string,
     public activityVersion: string,
-    public questions: Array<QuestionDefinition>
+    public questions: Array<QuestionDefinition>,
+    public displayOrder:number=1, public showActivityStatus = true
```

Comment:
can we use the new values from activity_definition

Reply:
I just added them as default values if the json doesn't have them but the column is selected

Reply:
@SimoneMaiwald If you think unnecessary I can remove

Reply:
if the json doesn't have them then default for showActivityStatus should be false

Reply:
fixed